pyproject.toml

```
@@ -373,6 +373,7 @@ markers = [
     "disable_qtimer_start: Disable timer start in this Test",
     "disable_qanimation_start: Disable animation start in this Test",
     "enable_console: Don't mock the IPython console (in QtConsole) in this Test",
+    "slow: mark a test as slow",
```

```suggestion
    # mark slow tests, so they can be skipped using: pytest -m "not slow"
    "slow: mark a test as slow",
```

I had to look up mark usage, so maybe this is helpful to add here?